### DISCUSSION TITLE
Zowe Explorer V2 Features

### DISCUSSION THREAD

**Discussion by jellypuno** [zowe/zowe-explorer-vscode#1414] [General]
Since we are all anticipating the Zowe V2 release, I would like to collect some ideas of the features that you would like to see in Zowe Explorer V2. I initially suggested the following features:

- Approved V1/V2 Conformance Criteria
- Improved Extensible APIs that conforms to our criteria
- Handling of Global and Project Level Team Config file
    - Loading for team-config created via CLI
    - Initializing a team-config file via Zowe Explorer
    - Securing credentials in the team-config file via Zowe Explorer **
- Multi-select support
- Zowe Explorer Templates (OMP Mentorship Program)

Please let me know if you have more features that you want to add in the list.

**Comment by jellypuno** [comment]
Breaking Changes that we want to implement
- https://github.com/zowe/vscode-extension-for-zowe/issues/1263
- Renaming tree nodes and settings
- Eclipse Che will be broken so research is needed here 

Onboarding will announce breaking changes for v2 will be announcedin Oct Zowe Webinar
Target Zowe V2 is Feb 2022 (Temporary)

**Reply by FALLAI-Denis** [reply]
For mvs, use "dataset" word and not "data set" expression.

**Reply by dkelosky** [reply]
I think IBM publications and ISPF use "data set" - in which context do you prefer "dataset" ?

**Reply by FALLAI-Denis** [reply]
Indeed, in the various IBM products and in the interfaces under ISPF, it is "data set" which is used and not "dataset" ...
As in [What is a data set?](https://www.ibm.com/docs/en/zos-basic-skills?topic=more-what-is-data-set)
So my request is not valid.

**Comment by jellypuno** [comment]
Being able to change mainframe password. APIML Squad is implementing this feature - Richelle to verify and open an issue.

**Reply by FALLAI-Denis** [reply]
Please, do not create dependencies with Zowe Server, (we do'nt use Zowe Server).

**Reply by JillieBeanSim** [reply]
is this available with z/OSMF APIs? If so a cli command/API could be implemented that could be consumed by Zowe Explorer.
